src/CMakeLists.txt

```diff
@@ -137,14 +137,17 @@ endfunction(zeth_test)
 zeth_test(test_simple test/simple_test.cpp TRUE)
 zeth_test(test_addition test/packed_addition_test.cpp TRUE)
 zeth_test(test_hex_to_field test/hex_to_field_test.cpp TRUE)
-zeth_test(test_note test/note_test.cpp TRUE)
-zeth_test(test_prfs test/prfs_test.cpp TRUE)
-zeth_test(test_commitments test/commitments_test.cpp TRUE)
+zeth_test(test_binary_operation test/binary_operation_test.cpp TRUE)
+# zeth_test(test_sha256 test/test_sha256.cpp TRUE)
```

Comment:
Why did you comment this one out @rrtoledo ?

Reply:
I added it because it was missing. 

You asked me to remove it as we do not use this circuit any longer.

Reply:
Not sure I understand, the test for sha is not ran here: https://github.com/clearmatics/zeth/blob/develop/src/CMakeLists.txt#L137-L147, so why do you add it and comment it?